src/NuGet.Indexing/OwnersResult.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Indexing
```

Comment:
got rid of nested classes. Public classes should not be nested. And one file per class